README.md

```diff
@@ -24,30 +34,33 @@ see https://neo4j.com/docs/operations-manual/current/backup/
 
 When working localy, docker-compose.override.yml adds a local bind mount to the /backups folder.
 
+The Crunchbase backup can be downloaded from Nexus. Decompress it in the ./backups bind mounted folder.
 
 Create an online backup:
 ```
-neo4j-admin backup --backup-dir /backups --database crunchbase-1.0.0
+docker-compose exec neo4j neo4j-admin backup --backup-dir /backups --database crunchbase-1.0.0
 ```
 
 Restore/create from dataset:
 ```
-neo4j-admin restore --from /datasets/4.2.0/crunchbase --database crunchbase-1.0.0  --verbose
+docker-compose exec neo4j neo4j-admin restore --from /datasets/4.2.0/crunchbase --database crunchbase-1.0.0  --verbose
 ```
 
 Restore from backup:
 ```
-neo4j-admin restore --from /backups/4.2.0/crunchbase --database crunchbase-1.0.0 --verbose
-```
-in v3.5.x
-```
-neo4j-admin restore --from /backups/3.5.15/fincrime-1.0.0/ --database graph.db
+docker-compose exec neo4j neo4j-admin restore --from /backups/4.2.0/crunchbase --database crunchbase-1.0.0 --verbose
```

Comment:
You dropped the example for 3.5 by accident here.

Reply:
As discussed together, this paragraph is confusing because the section is about Neo4j v4. I initially thought it was here by mistake. But it's there to avoid duplicating stuff in the previous Neo4j v3 section. So, let's revert that change. Things will get easier when we drop Neo4j v3 support.